bot/cogs/tags.py

```diff
@@ -92,8 +98,6 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
 
         `predicate` will be the built-in any, all, or a custom callable. Must return a bool.
         """
-        await self._get_tags()
```

Comment:
Since you are removing this line, consider changing the method back to non-async.

Reply:
Then everything looks good to go.